scripts/services/add-localizations.js

```diff
@@ -0,0 +1,249 @@
+const { promises: fs, existsSync, readFileSync } = require('fs');
```

Comment:
Before this a helper asynchronous function was used:
```
/**
 * Check if file is exist
 *
 * @param {string} filePath - path to the file
 * @returns {boolean} true - if exist, false - if does noe
 */
const isExist = async (filePath) => fs.access(filePath, fs.constants.F_OK)
    .then(() => true)
    .catch(() => false);

```
while check the existing files:
```
if (await isExist(translationFilePath)) {
                // Read and parse the translation content
                const translationContent = JSON.parse(await fs.readFile(translationFilePath));
                // Group translations by id, locale and use (name, description etc.)
                return groupFileContentByTranslations(translationContent, directory);
            }
            return null;
```
but because there is no much need to read files asynchronously I changed it to a synchronous, more readable version